### ISSUE TITLE
Please restore v1.004 JP glyphs

### ISSUE THREAD

**Issue by CoolMarvel43** [redchenjs/source-han-sans-classic#3]
Please reinstate some removed v1.004 JP glyphs, these files [from here](https://github.com/adobe-fonts/source-han-sans/issues/259#issuecomment-891759931) will help you review which glyphs to reinstate. I already sorted out all the JP glyphs needed to restore.

> With a little help from my friend on sorting through all the infos, here is a full list:
> [shs-v1.004-full-removed.pdf](https://github.com/adobe-fonts/source-han-sans/files/6923614/shs-v1.004-full-removed.pdf)
> [shs-1.004-full-removed.txt](https://github.com/adobe-fonts/source-han-sans/files/7025274/shs-1.004-full-removed.txt)

㐄㐪㐭㐯㑁㑒㑓㑞㑧㑹㒴㒶㓃㓜㓪㓬㔁㔓㔗㔜㔦㔰㔱㔳㔴㔵㔺㔼㕽㖖㖢㖫㗁㗔㗖㗫㘜㙄㙻㚆㚈㚉㚕㚻㛒㛷㜾㝇㝋㝑㝘㝙㝟㝠㝭㞪㞴㞷㞻㞼㟜㟟㟡㟧㟻㠉㠌㠎㠕㠝㠼㡞㡢㡣㡫㡬㡳㡽㢈㢋㢑㢕㢜㢢㢧㢴㣀㣃㣞㣠㣢㣪㤅㤠㤺㥀㥁㥐㥦㥩㥱㥲㦘㦛㦣㦥㦶㧈㧞㧨㨋㨗㨯㨻㪉㪘㪟㪰㪽㪿㫀㫤㫦㫨㫴㫷㬁㬊㬝㬥㬦㬧㬵㬶㭃㭕㭸㭻㮵㮻㯬㰓㰞㰭㰮㰱㰸㱅㱋㱝㲊㲝㳖㳙㳟㳵㳺㴀㴠㵒㵓㵧㵮㵶㵼㷱㸳㹣㹲㹹㼋㼘㼼㽪㾗㿂㿋㿤䀵䁀䁈䂊䂖䂢䂲䃍䃝䃾䄀䄂䄇䄦䅆䅥䆒䇊䇋䇒䇡䇱䈞䈧䈴䈵䉖䉜䉽䊅䊑䊢䊣䊯䊶䋁䋇䋐䋛䋨䌆䌇䌓䌖䌞䌬䍪䎩䏕䏬䑑䑧䑨䑭䒔䒢䒱䓐䓓䔊䔤䔵䔻䕌䕞䕶䖇䗙䗝䗟䘖䘘䘡䘱䘺䙞䙴䚈䚬䚱䚸䛑䛕䛦䛭䛮䛱䜋䜏䜢䜬䝉䝩䝺䝿䞯䠁䡱䢉䢗䢞䢟䢣䣛䣩䣴䤎䤛䤝䤞䤢䤣䤤䤩䤰䤱䤷䥃䥍䥎䥏䥢䥬䥮䥶䦇䦓䦚䦜䦾䧢䧶䩫䪄䪵䫓䫫䬢䭕䭰䮞䮲䯫䯬䰇䰌䰵䰺䱶䲬䲳䲾䴟䴡䶑䶬䶯䶰丽义乒乓仅优佢俖俷偝偮偯偻傇傼僁僗僠僽儮兝刽剎剢劲卟叻叼吆吔吙吣吷呯咂咓咔咣哋唒唿啋啕啥喌嗥嗦嗧嗨嘀嘟嘭噘嚉嚎嚐囆囔团垒垦垹垺埮堸塻壆奋奰孢孻实宪尘屉幂幙幠庴归忧怹恳悐惦愣慔懱执扰抚抰挎挕揍揞搣摁摵擏擫攎敥敹晱暯曞朌杹栠栦栺梑楧榵橗歶氋沎沷浂渃渪湡漜潕潹澫澿濿瀎瀱灱灴炋炓炣烅烆烌烍烒烳烴焟焨焬焺焻煂煄煍煏煔煘煟煰煼熁熂熼熽燆燊燤燨爃爅爏爞爣爧牊犡狅猼獛獳玤瑌盚眏眲瞁瞨矎磰磱礝礣稪稬竼笁笚笝筂筄筘筣筸箂箣篘篢篻篽篿簊簝簵粎糐罼肊肐肕胉胑胣脁脕脙脪脴腂腄腖腜腢腬膫臐舕艼芅芞芵苃苖苲苵苸苼茙茠茦茿荁荋荴莁莋莐莗莤莮莯菄菋菗龓鿅

**Exceptions**

Do not restore:

- 䂖 (looks weird)
- 䎩 (JP is 畄 not 甾)
- 䏕 (TW form good enough)
- 义 (looks weird)
- 垹 (TW form good enough)
- 屉 (世 looks weird)
- 归 (looks weird)
- 燊 (already restored in v2)

Restore and change:

- 䌓 (毎 change to 每)
- 䁈 (戸 change to 戶, with the JP detail shown on left)

<img width="281" alt="Screenshot 2021-09-29 at 01 56 29" src="https://user-images.githubusercontent.com/10541683/135140380-d05c4c5a-b06c-4a4b-8c08-916b900a4bcb.png">

Do not restore, but change yourself:

- 䈧 (爫 change to 爪)

Source file of 1.004 here:
https://github.com/adobe-fonts/source-han-sans/releases/tag/1.004
https://github.com/adobe-fonts/source-han-sans/releases/tag/1.004R

**Comment by NightFurySL2001** [comment]
@Marcus98T I dont think this project can be of any use (refer #1). I am currently working with a team to make a classical version of Source Han Sans, may I invite you to join us? You may contact me on nfsl-fonts@outlook.my.

**Comment by CoolMarvel43** [comment]
PM'ed you on that matter you mentioned, but I will keep the issue open regardless.
